### DISCUSSION TITLE
LNM as a FS2020 instrument

### DISCUSSION THREAD

**Discussion by kissandfly** [albar965/littlenavmap#1174] [Ideas]
Would be great to have a mod that replace Aera or G3x instrument for LNM. 
I am trying myself, but I can't make it clickable (not a coder)
![Capture d'écran 2024-07-31 092749](https://github.com/user-attachments/assets/1a3d52fa-260c-4d28-b47f-4c8a597b7397)

**Comment by albar965** [comment]
Good idea but I think you'd need a web developer for this. Maybe somebody else can volunteer.
